### PR TITLE
Feat/101 profile of owner

### PR DESCRIPTION
**Pull request by mirsadikov**
photo_2024-02-04 18 36 42 photo_2024-02-04 18 36 43

### CODE REVIEW

File: src/screens/explore/HostProfile/HostProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          </View>
+        ) : (
+          <Loader visible={loading} message="Loading host profile" />
+        )}
+      </ScrollView>
+    </ScreenTemplate>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (melievkh) [inline src/screens/explore/HostProfile/HostProfile.tsx]: better create seperate file for styles (for example, HostProfile.style.ts)

File: src/screens/explore/HostProfile/components/HostAbout.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          <Text style={styles.infoText}>{`Lives in: ${country}`}</Text>
+        </View>
+      </View>
+      <Text style={styles.description}>{description}</Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
```
Comment (melievkh) [inline src/screens/explore/HostProfile/components/HostAbout.tsx]: move all styles to a seperate file same as mentioned above

File: src/store/slices/hostProfileSlice.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    builder.addCase(AsyncThunks.getHostProfile.pending, onPending);
+    builder.addCase(AsyncThunks.getHostProfile.fulfilled, (state, action) => {
+      state.pending = false;
+      state.result = action.payload.data;
+    });
+    builder.addCase(AsyncThunks.getHostProfile.rejected, onError);
+  },
+});
+
+// export const hostProfileActions = hostProfileSlice.actions;
```
Comment (melievkh) [inline src/store/slices/hostProfileSlice.ts]: if it is not used you can remove

File: src/screens/profile/MyAccommodations/MyAccommodations.tsx [hunk 4]
```diff
@@ -72,7 +72,7 @@ const MyAccommodations = ({ navigation }: Props) => {
     dispatch(accommodationActions.clearError());
     dispatch(myAccommodationsListActions.clearError());
     fetchMyAccommodations();
-  }, [dispatch]);
+  }, [dispatch, fetchMyAccommodations]);
```
Comment (melievkh) [inline src/screens/profile/MyAccommodations/MyAccommodations.tsx:81]: isn't it causing infinite fetching when you add fetchMyAccommodations to dependency? better check on more time

**Review by melievkh — CHANGES_REQUESTED** [review]

File: src/screens/explore/HostProfile/HostProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    }
+  }, [dispatch, hostId, navigation]);
+
+  useEffect(() => {
+    fetchHostProfile();
+  }, [fetchHostProfile]);
+
+  return (
+    <ScreenTemplate>
+      <SafeAreaView>
```
Comment (melievkh) [inline src/screens/explore/HostProfile/HostProfile.tsx]: why we need SafeAreaView and StatusBar here? if there is a problem with Statusbar, it is not the best option

File: src/components/index.ts [hunk 6]
```diff
@@ -31,7 +31,7 @@ export {
 
 export * from './navigationLists';
 export * from './inputs';
-export * from './modals/bottomModals';
-export * from './modals/centerModals';
-export * from './modals/ExploreModals';
+export * from './Modals/bottomModals';
+export * from './Modals/centerModals';
```
Comment (melievkh) [inline src/components/index.ts]: better not to change filename, keep it as [modals] in everywhere and try "git config core.ignorecase false"
and make sure filename is changed in your branch also

**Review by melievkh — CHANGES_REQUESTED** [review]

File: src/screens/profile/MyAccommodations/MyAccommodations.tsx (hunk 4 above)
Comment (mirsadikov) [reply src/screens/profile/MyAccommodations/MyAccommodations.tsx:81]: no, I checked, there is noting that causes infinite fetching, function is wrapped with useCallback. Also this removes warning of useEffect missing dependency

**Review by melievkh — APPROVED** [review]

**Review by javokhirm — CHANGES_REQUESTED** [review]
there are conflicts

**Review by javokhirm — APPROVED** [review]

**Pull request merged**